products/eap/_common/product.yml

```diff
@@ -24,6 +24,23 @@ guides:
 default_download_artifact_type: installer
 description: 'An application platform for hosting your apps that provides an innovative modular, cloud-ready architecture, powerful management and automation, and world class developer productivity.'
 downloads:
+  7.0.0.Beta:
+    description: Major release of JBoss EAP 7.x series
+    tag_line: Featuring Java EE 7 Support. Try it today!
```

Comment:
Needs changing to:

Featuring support for Java EE 7, high performance Undertow web server, port reduction, and more. Try it today!